Roles/Neutral/Jester.cs

```diff
@@ -63,7 +66,11 @@ public override void CheckExile(NetworkedPlayerInfo exiled, ref bool DecidedWinn
         {
             if (isMeetingHud)
             {
-                name = string.Format(Translator.GetString("ExiledJester"), Main.LastVotedPlayer, Utils.GetDisplayRoleAndSubName(exiled.PlayerId, exiled.PlayerId, true));
+                if (RevealJesterUponEjection) = false
```

Comment:
cooked
if (Reveal.GetBool())

Reply:
Use this:
`if (RevealJesterUponEjection.GetBool())`